### PR TITLE
[FIX] Forecasted quantity for a make to order product

### PR DESCRIPTION
**Pull request by simongoffin**
The forecasted quantity for a make to order product is the quantity
in hand.

opw:627407

### CODE REVIEW

File: addons/procurement/procurement.py [hunk 1]
```diff
@@ -619,12 +619,6 @@ class product_template(osv.osv):
         'supply_method': 'buy',
     }
 
-class product_product(osv.osv):
-    _inherit="product.product"
-    _columns = {
-        'orderpoint_ids': fields.one2many('stock.warehouse.orderpoint', 'product_id', 'Minimum Stock Rules'),
```
Comment (Gorash) [inline addons/procurement/procurement.py]: Nein Achtung

File: addons/stock/product.py [hunk 2]
```diff
@@ -359,8 +359,17 @@ def _product_available(self, cr, uid, ids, field_names=None, arg=False, context=
             if f == 'outgoing_qty':
                 c.update({ 'states': ('confirmed','waiting','assigned'), 'what': ('out',) })
             stock = self.get_product_available(cr, uid, ids, context=c)
+            mto_ids = []
+            check = 'procure_method' in self._all_columns
```
Comment (Gorash) [inline addons/stock/product.py]: Strange to use this check.

Comment (simongoffin) [inline addons/stock/product.py]: In 7.0, several modules add the field 'procure_method', this is why we use a check (no way to overwrite _product_available in one module).

Comment (Gorash) [inline addons/stock/product.py]: ok

**Pull request closed without merging**